### DISCUSSION TITLE
Acronym resources

### DISCUSSION THREAD

**Discussion by kellijohnson-NOAA** [nmfs-ost/asar#106] [General]
Hi all, I thought it would be good to start a Discussion about resources that can be used to create an acronym list. There are obviously many acronyms used in fisheries leading to our acronym soup and those resources are spread out and often hard to find. So, if you know of an acronym list feel free to attach the resources in a comment below. Also @Schiano-NOAA feel free to delete this discussion if it is not appropriate here.

**Comment by kellijohnson-NOAA** [comment]
Endangered Species Act (ESA) Section 7(a)(2) Biological Opinion https://doi.org/10.25923/z5rd-xy02
List of acronyms is on pages 2--4.
[noaa_66645_DS1.pdf](https://github.com/user-attachments/files/18150520/noaa_66645_DS1.pdf)

**Comment by iantaylor-NOAA** [comment]
This is a very complete list at least for west coast groundfish: https://github.com/pfmc-assessments/sa4ss/blob/main/inst/rmarkdown/templates/sa/skeleton/sa4ss_glossaries.tex.

I'm curious what a good strategy will be for acronyms that have different definitions in different regions. I suppose we can just use different codes: `\gls{xyz}` vs `\gls{xyz1}` or if it's easier to remember, something like `\gls{xyz_NWFSC}`.
